packages/strapi-versioning-plugin/src/services/Versioning.ts

```diff
@@ -0,0 +1,50 @@
+import { Context } from "koa";
+
+type Entity = any;
+type Data = Array<Entity>;
+
+const isModelExists = (ctx: Context): boolean =>
+  (ctx.params !== undefined && ctx.params.model !== undefined) ||
+  Object.values(global.strapi.models).find(
+    (el) => el.collectionName == ctx.url.split("/")[1],
+  ) !== undefined;
+
+const findUid = (ctx: Context): string =>
+  ctx.params.model
+    ? global.strapi.db.getModel(ctx.params.model).uid
+    : Object.values(global.strapi.models).find(
+        (el) => el.collectionName == ctx.url.split("/")[1],
+      )!.uid;
```

Comment:
I rather would like to check is find returning anytning rather than using `find()!.uid`

Reply:
Api is handling this and if not exists it return 405